packages/e2e-tests/src/specs/wordpress/corsCheck.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import {
+  withExperimentalFeatures,
+  createNewStory,
+  activatePlugin,
+  deactivatePlugin,
+  takeSnapshot,
+} from '@web-stories-wp/e2e-test-utils';
+/**
+ * Internal dependencies
+ */
+import { addAllowedErrorMessage } from '../../config/bootstrap.js';
+
+describe('CORS check', () => {
+  withExperimentalFeatures(['enableCORSCheck']);
+  let removeCORSErrorMessage;
+  let removeResourceErrorMessage;
+
+  beforeAll(async () => {
+    await activatePlugin('e2e-tests-cors-error');
+    removeCORSErrorMessage = addAllowedErrorMessage(
+      'has been blocked by CORS policy'
+    );
+    removeResourceErrorMessage = addAllowedErrorMessage(
+      'Failed to load resource'
```

Comment:
Not exactly sure. But when I run this headless locally, i see this.

```
  ● CORS check › should see media dialog

    expect(jest.fn()).not.toHaveErrored(expected)

    Expected mock function not to be called but it was called with:
    ["Failed to load resource: net::ERR_FAILED"]

      at Object.assertExpectedCalls (../../../node_modules/@wordpress/jest-console/build/@wordpress/jest-console/src/index.js:36:4)
```

Reply:
@swissspidy are you happy with me leaving this in?

Reply:
It's important to know why a certain error is being suppressed / what it belongs to.

I just checked this myself to confirm that it's indeed the console error message for the `https://ps.w.org/web-stories/assets/banner-772x250.png` URL.

So that should be fine to leave it in.

Adding a comment explaining this would be helpful though.